shared/zkpp.js

```diff
@@ -114,3 +135,26 @@ export const buildRegisterSaltRequest = async (publicKey: string, secretKey: Uin
 
   return [contractSalt, base64ToBase64url(Buffer.concat([nonce, encryptedHashedPasswordBuf]).toString('base64'))]
 }
+
+export const buildUpdateSaltRequestEa = async (password: string, c: Uint8Array): Promise<[string, string]> => {
+  // Derive S_A and S_C as follows:
+  //   -> S_T -< BASE64(SHA-512(SHA-512(T) + SHA-512(c))[0..18]) with T being
```

Comment:
Please explain in the comment or by updating the variable names how `T` maps to the input parameters here. I'm assuming `c` in this comment refers to `c` in the parameter.

Reply:
`T` doesn't map at all to the input parameters.